lib/Find-VS2017.cs

```diff
@@ -228,9 +228,12 @@ private static bool CheckInstance(ISetupInstance2 setupInstance2, ref StringBuil
                     hasMSBuild = true;
                 else if (id == "Microsoft.VisualStudio.Component.VC.Tools.x86.x64")
                     hasVCTools = true;
-                else if (id.StartsWith(Win10SDKPrefix))
-                    Win10SDKVer = Math.Max(Win10SDKVer, UInt32.Parse(id.Substring(Win10SDKPrefix.Length)));
-                else if (id == "Microsoft.VisualStudio.Component.Windows81SDK")
+                else if (id.StartsWith(Win10SDKPrefix)) {
+                    string[] parts = id.Substring(Win10SDKPrefix.Length).Split('.');
+                    if (parts.Length == 2 && parts[1] != "Desktop")
```

Comment:
There is already a `Microsoft.VisualStudio.Component.Windows10SDK.15063.UWP.Native` that this does not catch